tests/e2e/operator/operator_suite_test.go

```diff
@@ -31,6 +38,11 @@ func TestInstall(t *testing.T) {
 
 func setup() {
 	GinkgoWriter.Println("************ Running Setup ************")
+
+	GinkgoWriter.Println("Initializing k8s client")
+	cl, err = k8sclient.InitK8sClient(kubeconfigPath)
```

Comment:
Why do we have to pass in kubeconfigPath? Can't the client look up the env var itself? I believe the controller-runtime client already does this automatically.